src/main/java/io/phdata/streamliner/schemadefiner/model/ColumnDiff.java

```diff
@@ -0,0 +1,152 @@
+package io.phdata.streamliner.schemadefiner.model;
+
+public class ColumnDiff {
+  private String previousName;
+  private String newName;
```

Comment:
I think we discussed using ColumnDef here. Are you planning to do that in this PR or another?

Reply:
what I understood from your last comment is because of these fields  isDeleted, isAdd, isUpdate we need this class.  
https://phdata.atlassian.net/browse/STREAM-15

Reply:
Gotcha, yeah I was saying that I think the ColumnDiff class can be:

ColumnDiff {
  ColumnDef prev;
  ColumnDef curr;
  bool isAdded;
  bool isDeleted;
  bool isUpdated;
}

I could be wrong but that seems to make sense in my mind.

Reply:
My mistake I could not understand. After looking at this class your comment in Jira ticket is clear now. 
I am thinking to make this change in next PR because I will need this PR changes for my next task like jackson for parsing.